src/06-loops.js

```diff
@@ -137,7 +230,25 @@ function firstIndexFound(array, target) {}
  *  lastIndexFound([ "left", "right", "left" ], "up");
  *  //> -1
  */
-function lastIndexFound(array, target) {}
+function lastIndexFound(array, target) {
```

Comment:
For the last three functions our indentation got a little flat.  Let's make sure to properly indent your code so other developers can see quickly where your code blocks begin and end.